apps/comments-ui/src/components/content/Comment.tsx

```diff
@@ -298,6 +298,32 @@ const CommentHeader: React.FC<CommentHeaderProps> = ({comment, className = ''})
 
         const element = (e.target as HTMLElement).ownerDocument.getElementById(comment.in_reply_to_id);
         if (element) {
+            const contentElement = element.querySelector('[data-testid="comment-content"]');
+            if (contentElement) {
+                // Find all paragraph elements
+                const paragraphs = contentElement.getElementsByTagName('p');
+                
+                Array.from(paragraphs).forEach((p) => {
+                    // Create a mark element for each paragraph
+                    const mark = document.createElement('mark');
+                    mark.className = 'animate-[highlight_2.5s_ease-out] [animation-delay:1s] bg-yellow-300/40 -my-0.5 py-0.5 dark:text-white/85 dark:bg-yellow-500/40';
+                    
+                    // Move the paragraph's content inside the mark
+                    while (p.firstChild) {
+                        mark.appendChild(p.firstChild);
+                    }
+                    p.appendChild(mark);
+
+                    // Remove the mark and restore original structure after animation
+                    mark.addEventListener('animationend', () => {
+                        while (mark.firstChild) {
+                            p.appendChild(mark.firstChild);
+                        }
+                        mark.remove();
+                    }, {once: true});
+                });
```

Comment:
Manipulating the DOM of a React-rendered component directly is generally a bad pattern because it can be wiped away at any point if something triggers a React render.

The likelihood of an unexpected render may be low enough in this case to not be too much of an issue but it would be good to not introduce discouraged patterns.

Reply:
Initial thought is adding a `highlightComment` action that changes a `commentToHighlight` property on our context to the comment's `id` value. Then the `Comment` component can change it's rendering to include the highlight classes.

Reply:
Certainly unlikely to have an unexpected render in this use case, but I think worth implementing a React implementation. I'll take a look at this. :)